packages/colors/src/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
     "extends": "../../../config/tsconfig.web",
     "compilerOptions": {
-        "outDir": "../lib/esm",
-    },
+        "outDir": "../lib/esm"
+    }
```

Comment:
Prettier `3.2.5` changed the default parser for `tsconfig.json` files from "jsonc" to "json"
  - JSON (strict) forbids trailing commas
  - JSONC (JSON with Comments) allows trailing commas and Prettier adds them by default

https://github.com/prettier/prettier/blob/main/CHANGELOG.md#use-json-parser-for-tsconfigjson-by-default-16012-by-sosukesuzuki